.eslintrc.js

```diff
@@ -1,58 +1,29 @@
-
 module.exports = {
-  "extends": ["eslint:recommended", "plugin:react/recommended"],
+  extends: ['eslint:recommended', 'plugin:react/recommended', 'prettier'],
```

Comment:
All the formatting was done by Prettier. It can be restored and file added to ignores if you wish.